<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1299</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grid Search Visualizer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="RunButton">
    <property name="geometry">
     <rect>
      <x>1055</x>
      <y>490</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>10</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QComboBox" name="AlgcomboBox">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>50</y>
      <width>82</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>DFS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BFS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dijkstra</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A*</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GBFS</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="algLabel">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Algorithm</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>30</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="CleanButton">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>490</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clean</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearButton">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>530</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>130</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Manhattan</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">heuristicRadioButtons</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>160</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Octal</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">heuristicRadioButtons</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>190</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Euclidian</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">heuristicRadioButtons</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>220</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Chebyshev</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">heuristicRadioButtons</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="heurLabel">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Heuristic</string>
    </property>
   </widget>
   <widget class="QLabel" name="InfoLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>661</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>1055</x>
      <y>250</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1055</x>
      <y>350</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Path Smoothig</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="smoothingV1But">
    <property name="geometry">
     <rect>
      <x>1055</x>
      <y>370</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>V1</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">pathSmoothingRadioButtons</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="smoothingV2But">
    <property name="geometry">
     <rect>
      <x>1055</x>
      <y>400</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>V2</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">pathSmoothingRadioButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="smoothButton">
    <property name="geometry">
     <rect>
      <x>1055</x>
      <y>530</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Smooth</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1299</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuSet_Grid_Size">
     <property name="title">
      <string>Set Grid Size</string>
     </property>
     <addaction name="action32x16"/>
     <addaction name="action64x32"/>
     <addaction name="action128x64"/>
    </widget>
    <widget class="QMenu" name="menuNeighbours_Configuration">
     <property name="title">
      <string>Neighbours Configuration</string>
     </property>
     <addaction name="action4_Neighbours"/>
     <addaction name="action8_Neighbours"/>
    </widget>
    <addaction name="menuSet_Grid_Size"/>
    <addaction name="menuNeighbours_Configuration"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action32x16">
   <property name="text">
    <string>32x16</string>
   </property>
  </action>
  <action name="action64x32">
   <property name="text">
    <string>64x32</string>
   </property>
  </action>
  <action name="action4_Neighbours">
   <property name="text">
    <string>4 Neighbours</string>
   </property>
  </action>
  <action name="action8_Neighbours">
   <property name="text">
    <string>8 Neigbours</string>
   </property>
  </action>
  <action name="action128x64">
   <property name="text">
    <string>128x64</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="heuristicRadioButtons"/>
  <buttongroup name="pathSmoothingRadioButtons"/>
 </buttongroups>
</ui>
